tasks/pipeline.py

```diff
@@ -522,7 +525,10 @@ def changelog(ctx, new_commit_sha):
         if "dependabot" in author_email or "github-actions" in author_email:
             messages.append(f"{message_link}")
             continue
-        author_handle = ctx.run(f"email2slackid {author_email.strip()}", hide=True).stdout
+        if author_email in EMAIL_SLACK_ID_MAP:
+            author_handle = EMAIL_SLACK_ID_MAP[author_email]
+        else:
+            author_handle = ctx.run(f"email2slackid {author_email.strip()}", hide=True).stdout.strip()
```

Comment:
do we need to do any error handling for email2slackid in case someone else doesn't have an email?

Reply:
the output is blank if there is no result, the if statement just below this line handles that case and instead we write their email to the output message

Reply:
I'm not sure there is another case we are not handling, like if the SDM api is down. I could probably check for `sterr` here and default to user email in that case too, but this will likely have the same result (stout being blank)

Reply:
If the script has a non-zero exit code, the entire task fails.